### DISCUSSION TITLE
pools.2021-June-4 succesfully deployed. (I think)

### DISCUSSION THREAD

**Discussion by LouisLoudogTrottier** [Chia-Network/chia-blockchain#6491] [Show and tell]
How do we know everything is working up to where the dev are for the pool protocol?

I have followed (more or less) these steps https://github.com/Chia-Network/pool-reference#install-and-run-testnet and after resolving an issue https://github.com/Chia-Network/pool-reference/issues/44 i was able to create a singleton, plot and receive 0.25txch in the pool wallet.

See on the left, the 1st keys generated and used as `default_target_puzzle_hash` is where i received the 0.25 txch
![Screenshot from 2021-06-05 18-37-56](https://user-images.githubusercontent.com/56370085/120909176-6636b500-c640-11eb-91af-eb290e974c8d.png)

I have a few questions:
- The pool-reference readme says «Note that claiming rewards and switching pools are still not enabled,». From what i understoond from the zoom for pool operator, someone needs to claim the reward in order to send everyone's shares. ... So the question is, The fact that i've received 0.25 txvh in the pool wallet means i have accomplished what it is coded to do so far?

- Is it normal that the 0.25 whent to the pool and not the farmer? I tought 0.25 was goin to the farmer and the rest to the pool. I did not receive anything in the `pool_fee_puzzle_hash` wallet, but it is not fully synced yet.

- What would be the next step like? Can i keep building on what i got a keep ploting or should i wait for the next update? How will the update process be like? As simple as updating the chia-blockchian repo and run the install from the pool-reference repo?

- If i am right about succefully deploy the code as it is so far, how can i assist you? I would be willing to help testing and report glitch and bug. I have written all the step i took to get where i am and willing to provide logs and documentation if needed.

Regards.

--- 
 
UPDATE : I have received anothe 1 xtch   to the `default_target_puzzle_hash` wallet. 
![Screenshot from 2021-06-05 23-38-59](https://user-images.githubusercontent.com/56370085/120911689-3f37ad80-c657-11eb-976f-657d5d2fb38a.png)
- How to check if it is the same reward or if it was just the wallet not fully synced or the reward splitted in smaller chunk?

**Comment by mariano54** [comment]
The reward is split into two: 0.25XCH for the farmer and 1.75XCH for the pool. The 0.25 is paid out automatically by the blockchain into the user's wallet.
The 1.75 goes to the p2_singleton_puzzle_hash. This must be claimed by the pool. Then the pool will distribute rewards to each user.
We're not providing user support at this moment, only pool operator / developer support.

**Reply by LouisLoudogTrottier** [reply]
I'm a moderator on a facebook gorup of ~7k members and saw people asking about the pools. So i got into the idea of seeing what i could do but seems i will certainly deploy a pool as soon as i got everything i need to do so the right way. I've been working with servers for the last 10 years and have really good knowledge of cryptocurrency. This is all full and games for me.

Best regards, thanks you very much for your time and work.
